src/JWT/JWT.cs

```diff
@@ -239,10 +239,12 @@ public static void Verify(string payloadJson, string decodedCrypto, string decod
         {
             if (decodedCrypto != decodedSignature)
             {
-	            var signatureVerificationException = new SignatureVerificationException("Invalid signature");
-				signatureVerificationException.Data.Add("Expected", decodedCrypto);
-				signatureVerificationException.Data.Add("Received", decodedSignature);
-	            throw signatureVerificationException;
+               var signatureVerificationException = new SignatureVerificationException("Invalid signature")
```

Comment:
I think that one good enough, especially if we'll reside the helper method here. Pelsse go ahead and rebase, I'll merge